src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -195,15 +203,18 @@ COMMIT TRANSACTION
                         SELECT Value, QuantityCodeId FROM dbo.QuantityCode";
 
                     string commaSeparatedResourceTypes = string.Join(",", ModelInfoProvider.GetResourceTypeNames());
-                    string searchParametersJson = JsonConvert.SerializeObject(_searchParameterDefinitionManager.AllSearchParameters.Select(p => new { Name = p.Name, Uri = p.Url }));
                     string commaSeparatedClaimTypes = string.Join(',', _securityConfiguration.PrincipalClaims);
                     string commaSeparatedCompartmentTypes = string.Join(',', ModelInfoProvider.GetCompartmentTypeNames());
+                    IEnumerable<ResourceSearchParameterStatus> statuses = _filebasedSearchParameterStatusDataStore.GetSearchParameterStatuses().Result;
```

Comment:
Prefer to use `.GetAwaiter().GetResult()` instead of `.Result`

Reply:
Thanks, fixed in acc8ad2.

Just to confirm, is this to avoid an `AggregateException`? Did a quick search and found [this](https://stackoverflow.com/questions/17284517/is-task-result-the-same-as-getawaiter-getresult).

Reply:
Yeah, I think synchronous code generally doesn't expect AggregateExceptions